.docs/content/2.guide/how-to-troubleshoot

```diff
@@ -0,0 +1,81 @@
+# How to troubleshoot
+
+Things can go wrong. As a developper, you know this already. But what to do when something goes wrong in ArmoniK ?
+
+This section will give you tips to check and fix problems for those times when it just won’t work and you do not know why.
+___
+## Your deployment failed or is malfunctioning
+
+First, note that if you deployment is local, MongoDB and Redis certifcates are only available for 7 days. Past this period, your deployment might encounter issues.
+Our suggestion is to destroy your deployment and redeploy.
```

Comment:
In the new version of the Infrastructure, the user can set the certificate validity parameter before the deployment, in the parameters.tfvars. Otherwise, the default validity is set to 8760 hours (one year).

Reply:
are you talking about the upcoming version or the one in place actually?

Reply:
upcoming version: infra=0.1.0